core/lcsb/rems.py

```diff
@@ -81,22 +102,36 @@ def handle_rems_entitlement(data: Dict[str, str]) -> bool:
     expiration_date = data.get('end')
 
     if expiration_date is None:
-        expiration_date = date.today() + timedelta(
-            days=getattr(settings, 'ACCESS_DEFAULT_EXPIRATION_DAYS', 90)
-        )
+        expiration_date = build_default_expiration_date()
     else:
         expiration_date = isoparse(expiration_date).date()
 
-    logger.debug(f'REMS :: * data access request id: {application}, user_oidc_id: {user_oidc_id}, user_email: {email}, resource: {resource}')
+    logger.debug(
+        f'REMS :: * data access request id: {application}, user_oidc_id: {user_oidc_id}, user_email: {email}, resource: {resource}')
```

Comment:
This looks like a black-linted line. Ending parenthesis should be on a new line I think

Reply:
I've reformatted the whole file with latest black